client/src/game/ui/menu/menu.vue

```diff
@@ -68,6 +68,25 @@
                                 <div v-if="!notes.length">No notes</div>
                             </div>
                         </div>
+                        <!-- MARKERS -->
+                        <button class="menu-accordion">Markers</button>
+                        <div class="menu-accordion-panel">
+                            <div id="menu-markers">
+                                <table width="100%">
+                                    <tbody>
+                                        <tr v-for="marker in markers" :key="marker.uuid" style="cursor:pointer">
+                                            <td style="white-space: nowrap;" width="99%" @click="jumpToMarker(marker)">
+                                                {{ marker.name || "[?]" }}
+                                            </td>
+                                            <td style="white-space: nowrap;" @click="delMarker(marker)">
+                                                <i class="far fa-minus-square"></i>
+                                            </td>
+                                        </tr>
+                                    </tbody>
+                                </table>
```

Comment:
I'm not super fond of <tables> :p If you can, rework it to a flexbox or grid, if you're not that familiar with it, I don't mind doing this as it shouldn't take a lot of effort

Reply:
Maybe not but this was twofold: firstly, when I used your approach, when making a lot of markers and notes, the menu bar gets a scroll bar. Scrolling would make the text and the icons move independently from each other (it still happens with the + for the notes). Using tables created a more 'sturdy' design which moved them together. It might have to do with the width that changes when adding the scrollbar but didn't want to mess with it too much.
Secondly it feel this gives the programmer a slightly more clear view about the design rather than having a lot of divs and having to figure out which class does what. But that's entirely personal.

Reply:
Using css grid is just simpler imo.  The 99% width and nowrap whitespace stuff feels wrong.

There is no absolute right/wrong here as this is just my preference ;)